network/network.go

```diff
@@ -68,9 +62,7 @@ type Network struct {
 	lastTransactionTracker lastTransactionTracker
 	protocols              []transport.Protocol
 	connectionManager      transport.ConnectionManager
-	graph                  dag.DAG
-	publisher              dag.Publisher
-	payloadStore           dag.PayloadStore
+	txState                dag.State
```

Comment:
why `txState` (vs. just `state` or `dagState`, since the package is called DAG)?

Reply:
renamed it to `state` in all packages outside `dag`